test/modules/losses/test_contrastive_loss_with_temperature.py

```diff
@@ -53,6 +53,20 @@ def test_local_loss(self):
         self.assertEqual(loss.size(), torch.Size([]))
         self.assertAlmostEqual(loss.item(), 9.8753, 3)
 
+    def test_temperature_clamp(self):
```

Comment:
nit: this is really just testing that the existing clamping works as expected. Would be nice to instead explicitly pass values for logit_scale_min/max here